src/screens/history/HistoryScreen.tsx

```diff
@@ -71,6 +71,23 @@ const HistoryScreen = ({ navigation }: HistoryScreenProps) => {
     setFalse: closeClearHistoryDialog,
   } = useBoolean();
 
+  useEffect(
+    () =>
+      navigation.addListener('tabPress', e => {
+        let lastNovel = history[history.length - 1];
+        if (navigation.isFocused() && lastNovel) {
+          e.preventDefault();
+
+          navigation.navigate('Novel', {
+            name: lastNovel.name,
+            path: lastNovel.novelPath,
+            pluginId: lastNovel.pluginId,
+          });
+        }
+      }),
+    [navigation],
+  );
```

Comment:
press on history while in history = open last read novel's overview page
mihon opens the next chapter, but that seemed like pain and this makes sense to me

Reply:
mihon history resume logic is flawed in that regard